addons/account/models/chart_template.py

```diff
@@ -1154,9 +1154,12 @@ class AccountReconcileModelTemplate(models.Model):
     label = fields.Char(string='Journal Item Label')
     amount_type = fields.Selection([
         ('fixed', 'Fixed'),
-        ('percentage', 'Percentage of balance')
+        ('percentage', 'Percentage of balance'),
+        ('regex', 'From label'),
         ], required=True, default='percentage')
     amount = fields.Float(string='Write-off Amount', digits=0, required=True, default=100.0, help="Fixed amount will count as a debit if it is negative, as a credit if it is positive.")
+    amount_from_label_regex = fields.Char()
+    decimal_separator = fields.Char(help="Every character that is nor a digit nor this separator will be removed from the matching string")
```

Comment:
can't you just ignore `,` and `.` all the time and let parseFloat recognize thousand separators and decimal points?